src/utils/templates.ts

```diff
@@ -1,12 +1,14 @@
 import joplin from "api";
-import { getAllNotesInFolder } from "./folders";
+import { getAllNotesInFolder, Note } from "./folders";
 
-export const getUserTempateSelection = async (templatesFolderId: string): Promise<string | null> => {
+type NoteProperty = "body" | "id" | "title";
+
+export const getUserTempateSelection = async (templatesFolderId: string, property: NoteProperty = "body"): Promise<string | null> => {
```

Comment:
Consider fixing the typo in the function name `getUserTempateSelection` -> `getUserTemplateSelection`

Reply:
done